config/locales/en/placements/schools/placements.yml

```diff
@@ -32,3 +72,4 @@ en:
           remove_placement: Remove placement
         destroy:
           placement_removed: Placement removed
+
```

Comment:
`bin/lint` should remove this

Reply:
Weirdly it did not, I have manually corrected.